### PR TITLE
Java11 updates

### PR DESCRIPTION
**Pull request by smlambert**
- updates to enable Java11 testing

### CODE REVIEW

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -26,10 +25,15 @@ def stageTestBinaries() {
 }
 
 def setupEnv() {
+	// env.JAVA_VERSION = "${getJavaVersionFromJvmVersion()}"
```
Comment (smlambert) [inline buildenv/jenkins/JenkinsfileBase]: Leaving this as comment, as I wish to return to it next week in a separate PR

File: buildenv/jenkins/openjdk_x86-64_windows [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!groovy
+LABEL='hw.arch.x86&&sw.os.win'
```
Comment (llxia) [inline buildenv/jenkins/openjdk_x86-64_windows]: Should we use `sw.os.windows` according to https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/93?
Eclipse OpenJ9 is already updated to use `sw.os.windows`

**Comment by llxia** [comment]
Approve the change so we can get generic files in. wins file is not used in Eclipse Openj9, so it can be addressed later.

File: buildenv/jenkins/openjdk_x86-64_windows (hunk 2 above)
Comment (smlambert) [reply buildenv/jenkins/openjdk_x86-64_windows]: Yes we should use sw.os.windows, I will update and merge.  Thanks!

**Comment by smlambert** [comment]
Since I have not removed any of the versioned data jenkinsfiles, all the old builds will work as is (using the old files).  We decided to push this in now, before teh getJavaVersionFromJvmVersion() method is completely working, but I left the commented line in, as I plan to return to it this week, because then we could remove all of the versioned data jenkinsfiles and not have to make any updates to job config.

**Pull request merged**